cmd/gokeyless/gokeyless.go

```diff
@@ -19,18 +16,24 @@ import (
 )
 
 var (
-	port        string
-	metricsPort string
-	initCert    bool
-	certFile    string
-	keyFile     string
-	caFile      string
-	keyDir      string
-	pidFile     string
+	initToken    string
+	initCertFile string
+	initKeyFile  string
+	initEndpoint string
+	port         string
+	metricsPort  string
+	certFile     string
+	keyFile      string
+	caFile       string
+	keyDir       string
+	pidFile      string
 )
 
 func init() {
-	flag.BoolVar(&initCert, "init", false, "Initialize new server authentication key and certificate")
+	flag.StringVar(&initToken, "init-token", "", "API token used for server initialization")
+	flag.StringVar(&initCertFile, "init-cert", "default.pem", "Default certificate used for server initialization")
+	flag.StringVar(&initKeyFile, "init-key", "default-key.pem", "Default key used for server initialization")
+	flag.StringVar(&initEndpoint, "init-endpoint", "https://apix.cloudflare.com/certificate", "API endpoint for server initialization")
```

Comment:
Good point. We should make sure to update it before the gokeyless 0.4 tagged release/package builds.